specs/cli/cli.md

```diff
@@ -0,0 +1,55 @@
+# FAST CLI (WIP)
```

Comment:
I would expect this to be more of a Proposed Issue, rather than a PR or expect this to be included as part of the implementation PR.